universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -51,6 +51,10 @@ public void setPreferredLocale(Locale locale) {
     this.preferredLocale = locale;
   }
 
+  public boolean hasPath(Path path) {
```

Comment:
Suggestion (non-blocking): there's a distinction between the underlying JSON data containing a path and it containing a value at that path i.e. the value at the path could be `null`. Recommend renaming this method to `hasValueAtPath`

Reply:
Good point - updated method name